src/analytics/private/sql-def.ts

```diff
@@ -12,51 +12,44 @@
  */
 
 import { SQLDef } from './model';
+import { CLICKSTREAM_DEVICE_VIEW_NAME, CLICKSTREAM_EVENT_PARAMETER_VIEW_NAME, CLICKSTREAM_EVENT_VIEW_NAME, CLICKSTREAM_LIFECYCLE_DAILY_VIEW_NAME, CLICKSTREAM_LIFECYCLE_WEEKLY_VIEW_NAME, CLICKSTREAM_RETENTION_VIEW_NAME, CLICKSTREAM_SESSION_VIEW_NAME, CLICKSTREAM_USER_ATTR_VIEW_NAME, CLICKSTREAM_USER_DIM_VIEW_NAME } from '../../common/constant';
 
 export const reportingViewsDef: SQLDef[] = [
   {
     updatable: 'false',
-    sqlFile: 'clickstream_event_view.sql',
+    sqlFile: `${CLICKSTREAM_EVENT_VIEW_NAME}.sql`,
```

Comment:
We can add `viewName` instead of the file name.

Reply:
replace sqlFile?

Reply:
add viewName property